src/Symfony/Component/Validator/Tests/Constraints/FileTest.php

```diff
@@ -31,6 +31,22 @@ public function testMaxSize($maxSize, $bytes, $binaryFormat)
         $this->assertTrue($file->__isset('maxSize'));
     }
 
+    /**
+     * @dataProvider provideValidSizesForIntegerOverflow
+     */
+    public function testMaxSizeWithIntegerOverflow($maxSize, $bytes, $binaryFormat)
+    {
+        if (32 !== 8 * \PHP_INT_SIZE) {
```

Comment:
```suggestion
        if (4 !== \PHP_INT_SIZE) {
```

(we do this in other places too)